lib/Serverless.test.js

```diff
@@ -104,6 +109,10 @@ describe('Serverless', () => {
       expect(serverless.classes.Service).to.deep.equal(Service);
     });
 
+    it('should store the ConfigSchemaHandler class inside the classes object', () => {
+      expect(serverless.classes.ConfigSchemaHandler).to.deep.equal(ConfigSchemaHandler);
+    });
```

Comment:
In case of this file, it's totally fine to leave those tests as you prepared them (no point to refactor it now)